packages/react-native/ReactAndroid/src/main/java/com/facebook/react/modules/systeminfo/AndroidInfoHelpers.kt

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.modules.systeminfo
+
+import android.content.Context
+import android.os.Build
+import com.facebook.common.logging.FLog
+import com.facebook.react.R
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.nio.charset.Charset
+import java.util.Locale
+
+public object AndroidInfoHelpers {
+
+  public const val EMULATOR_LOCALHOST: String = "10.0.2.2"
+  public const val GENYMOTION_LOCALHOST: String = "10.0.3.2"
+  public const val DEVICE_LOCALHOST: String = "localhost"
+  public const val METRO_HOST_PROP_NAME: String = "metro.host"
+  private val TAG = AndroidInfoHelpers::class.java.simpleName
+  private var metroHostPropValue: String? = null
+
+  private fun isRunningOnGenymotion(): Boolean = Build.FINGERPRINT.contains("vbox")
+
+  private fun isRunningOnStockEmulator(): Boolean = Build.FINGERPRINT.contains("generic") or Build.FINGERPRINT.startsWith("google/sdk_gphone")
+
+  public fun getServerHost(port: Int): String = getServerIpAddress(port)
+
+  public fun getServerHost(context: Context): String = getServerIpAddress(getDevServerPort(context))
+
+  public fun getAdbReverseTcpCommand(port: Int): String = "adb reverse tcp:$port tcp:$port"
+
+  public fun getAdbReverseTcpCommand(context: Context): String = getAdbReverseTcpCommand(getDevServerPort(context))
+
+  public fun getFriendlyDeviceName(): String {
```

Comment:
Add @JvmStatic here (and to all others potentially) to keep this backwards compatible.

Reply:
Understood 👍 will update.